feat(EuiSuperUpdateButton): support custom button labels via `children` prop (#7576)
elastic/eui: changelogs/upcoming/7576.md

> - Added support for custom `EuiSuperUpdateButton` labels by passing an optional `children` prop

Reviewer: ```suggestion
- Updated `EuiSuperUpdateButton` to support custom button text via an optional `children` prop
```

Reviewer: We def haven't been consistent in the past, in general what I've been doing is using `Added` for totally new components and `Updated` for existing components, although it's def more a guideline than a rule
